src/error.rs

```diff
@@ -18,18 +17,10 @@ pub enum Error {
     #[error("encoding: {0}")]
     Encoding(String),
 
-    /// Metadata was expired.
-    #[error("expired {0} metadata")]
-    ExpiredMetadata(Role),
-
     /// An illegal argument was passed into a function.
     #[error("illegal argument: {0}")]
     IllegalArgument(String),
 
-    /// The metadata was missing, so an operation could not be completed.
-    #[error("missing {0} metadata")]
-    MissingMetadata(Role),
-
     /// There were no available hash algorithms.
```

Comment:
Perhaps not too big of a concern, but I believe those two errors will re-appear on a later date...

Reply:
Thank you! I will definitely keep this issue in mind when working on this project and make sure MissingMetadata is included in Error handling.